ie: replace the geometry from one imagery with another (fixes #2176)

amandasaurus opened the pull request.

matheusgomesms commented:
Thanks, Amanda!

Do you think we could add a license_url and a privacy_policy_url in this layer?

grischard commented:
@amandasaurus would the polygon from http://download.geofabrik.de/europe/ireland-and-northern-ireland.html work?

amandasaurus commented:
I've rebased this PR, and added the `privacy_policy_url`. However, I'm unsure for the `license_url`. The original maps are out of copyright (being from the 1940s). I don't know any more.

andrewharvey commented:
I think it would be good then if the `license_url` could point to any metadata or source page for the original maps, ideally which then shows some information we can use to verify/determine they are out of copyright.
